website/docs/d/kube_config.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_kube_config"
+sidebar_current: "docs-data-source-vcfa-kube-config"
+description: |-
+  Provides a data source to fetch the kubeconfig data from VMware Cloud Foundation Automation.
+---
+
+# vcfa\_kube\_config
```

Comment:
Shall it be `vcfa_kube_config` or `vcfa_kubeconfig`? I see in their docs (even in ours, paragraph below this title) that it's always referred as kubeconfig (not separated).

Probably a nitpick, but better now than never 😄 

cc: @frodenas

Reply:
Good catch @adambarreiro. Yes, it's mostly referred as `kubeconfig`, so better to rename the datasource to `vcfa_kubeconfig`.

Reply:
Changed. Tests are still passing.